### PR TITLE
fix the double entries all years in selectbox

### PR DESCRIPTION
**Pull request by inpercima**
* reorder filterType to pass filterYears
* use 'all' instead off empty ('') for all years

### CODE REVIEW

File: src/main/resources/public/partials/activities.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <input class="form-control" type="number" data-ng-change="vm.list()" data-ng-model="vm.size" placeholder="Resultset size" id="size" min="0" />
   </div>
   <div class="form-group form-inline">
-    <select class="form-control" data-ng-disabled="vm.allActivityTypes" data-ng-change="vm.list()" data-ng-model="vm.filterType" id="filterType" data-ng-options="filterType.type for filterType in vm.types track by filterType.key" multiple></select>
+    <select class="form-control" data-ng-change="vm.list()" data-ng-model="vm.filterType" id="filterType" data-ng-options="filterType.type for filterType in vm.types track by filterType.key" data-ng-disabled="vm.allActivityTypes" multiple></select>
```
Comment (sepe81) [inline src/main/resources/public/partials/activities.html]: Why did you change the order?

Comment (inpercima) [inline src/main/resources/public/partials/activities.html]: To have the same order like the other selectboxes.

**Comment by inpercima** [comment]
Ready to check, typo fixed.

**Comment by inpercima** [comment]
Would you merge?
